WcaOnRails/db/migrate/20160303144700_fill_teams_tables.rb

```diff
@@ -0,0 +1,39 @@
+class FillTeamsTables < ActiveRecord::Migration
+  def change
+    execute <<-SQL
+      INSERT INTO teams (friendly_id, name, description) values ('results','Results Team','This team is responsible for managing all competition results.')
+    SQL
+
+    execute <<-SQL
+      INSERT INTO teams (friendly_id, name, description) values ('software','Software Team','This team is responsible for managing the WCA\'s software (website, scramblers, workbooks, admin tools).')
+    SQL
+
+    execute <<-SQL
+      INSERT INTO teams (friendly_id, name, description) values ('wdc','Disciplinary Committee','This committee advises the WCA Board in special cases, like alleged violations of WCA Regulations, and may be contacted by WCA members in case of important personal matters regarding WCA competitions.')
+    SQL
+
+    execute <<-SQL
+      INSERT INTO teams (friendly_id, name, description) values ('wrc','Regulations Committee','This committee is responsible for managing the WCA Regulations.')
+    SQL
+
+    execute <<-SQL
+      INSERT INTO team_members (team_id, user_id, start_date) SELECT teams.id, users.id, Date.today FROM teams,users WHERE teams.friendly_id = 'results' and users.results_team = '1'
```

Comment:
Also, I intend to go back and search for the actual start_dates.
Not sure if it makes sense to include that here, that's why I'm just using NOW()